week-2/L-array-methods-2/exercise2.js

```diff
@@ -7,8 +7,12 @@
   Tip: use the string method .split() and the array method .join()
 */
 
-function capitalise(str) {}
-
+function capitalise(str) {
+  var character1 = str.split("");
```

Comment:
`letters` would be a more descriptive name for this variable.

Otherwise, good solution!